# entity_name provides the name of the robot as is presented to the context broker by firos
string entity_name

# entity_status provides the status of the robot in the context broker: CREATED=1 or REMOVED=0
int16 entity_status

int16 CREATED=1
int16 REMOVED=0
